code-studio/build-commands.js

```diff
@@ -49,10 +49,14 @@ exports.browserify = function (config) {
   var command = (shouldWatch ? 'watchify -v' : 'browserify') +
     (shouldMinify ? '' : ' --debug');
 
+  var reactifyStep = '-t reactify';
+
+  var globifyStep = '-t require-globify';
+
   var factorStep = '';
   if (shouldFactor) {
     factorStep = "-p [ factor-bundle -o '" + (shouldMinify ? 'uglifyjs ' : '') +
-      '>' + buildPath + "`basename $FILE .js`" + extension + "']";
+      '>' + buildPath + "`dirname $(realpath --relative-to " + srcPath + " $FILE)`/`basename -s .js -s .jsx $FILE`" + extension + "']";
```

Comment:
I wonder if this is complex enough that we should (a) just make a simple bash script that we call or (b) create a simple bash function in an earlier step, then call it here. Thoughts?

At the very least, it's probably worth adding a comment to the code explaining what's happening.